### PR TITLE
Personal Portfolio Mobile:

### PR DESCRIPTION
**Pull request by Sofinaye**
Personale Portfolio Mobile:
   In this Personale portfolio mobile I:
      section of projects I have done.
      selection of Technical skills for the job.
      contact form that leads to reaching me.

### CODE REVIEW

**Comment by Sofinaye** [comment]
pull request for portfolio mobile

**Review by topeogunleye — COMMENTED** [review]
Hi @Sofinaye, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project, that doesn't have a pull request description, kindly add one. See a demo on how to add a PR description below:

pull_request_desc
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

**Comment by Sofinaye** [comment]
In this Personale portfolio mobile I:
 section of projects I have done.
 selection of Technical skills for the job.
 contact form that leads  reaching me.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+        <ul class="navbar-list">
+          <li class="navbar-item">Portofolio</li>
+          <li class="navbar-item">About</li>
+          <li class="navbar-item">Contact</li>
+          <li class="navbar-item"><img src="./img/666162.png" alt="" class="nav-icon"></li>
+        </ul>
+      <div class="navbar-toggle">
+        <i class="fa-solid fa-bars"></i>
+      </div>
+      </nav>
```
Comment (Qoosim) [inline index.html]: - [x] It  is required to use `<header>` tag here as a container for this section for semantic reason. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. You can check [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. Kindly change the generic `<div>` tag to `<header>` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;400;500;700;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="wrapper">
+    
+    
+      <nav class="navbar">
+
+        <div class="navbar-brand">Abenezer</div>
```
Comment (Qoosim) [inline index.html]: - [x] It is a universal standard that the logo on a website should be a link to the homepage of the site. This is for three reasons: convention, semantics, and user experience(UX). Kindly add a link to your title. e.g `<div class="navbar-brand"><a href="home-page-link">Abenezer</a></div>`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+    
+      
+        <ul class="navbar-list">
+          <li class="navbar-item">Portofolio</li>
+          <li class="navbar-item">About</li>
+          <li class="navbar-item">Contact</li>
+          <li class="navbar-item"><img src="./img/666162.png" alt="" class="nav-icon"></li>
```
Comment (Qoosim) [inline index.html]: - [ ] It is required that the hamburger of your page should be wrapped in a `<nav>` tag as this for semantics reason, and the right position for convention and user experience for mobile devices.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+      <div class="body-text-main">
+        <p>I can help you build a product,feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      </div>
+      <ul class="portfolio-social">
+        <li class="portfolio-social-item"><i class="fa-brands fa-github"></i></li>
+        <li class="portfolio-social-item"><i class="fa-brands fa-linkedin-in"></i></li>
+        <li class="portfolio-social-item"><i class="fa-solid fa-hand-peace"></i></li>
+        <li class="portfolio-social-item"><i class="fa-brands fa-twitter"></i></li>
+        <li class="portfolio-social-item"><i class="fa-solid fa-m"></i></li>
+      </ul>
```
Comment (Qoosim) [inline index.html]: - [x] It is required that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Kindly add `a` tag to every of your social icons.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+      </section>
```
Comment (Qoosim) [inline index.html]: - [x] There is a background image for section of your portfolio which is supposed to be added with css background-image property. At this moment, that background image is missing on your project. Kindly add the background image to your headline section.

### Your Design.
your-design

### Figma Design
figma-design

- [x] Also, it is required to stick to the guiding template as much as possible. This includes font, color, background-color, background-image, margin, padding among others. Kindly increase the font-size of your `p` tag content and make sure you use the same font-family for the section. This should be applied to every other tags on this section.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Sofinaye, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  You wrote good README file, but not professional. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

# ===== Important Information =====

**NOTE:**  _**You PR includes sections that are out of the scope of the project milestone requirements and are not going to be reviewed. You're therefore asked to remember to provide a link to the additional code in the Issue you would create for the next milestone (as everything is already done, you won't have any changes to open a Pull Request). You will need to create a link(s) to specific commit or line(s) of code in Pull Request and include that link in the description of the Issue your created for the corresponding milestone.**_

File: index.html (hunk 3 above)
Comment (Sofinaye) [reply index.html]: @Qoosim 
The hamburger is already wrapped in the <nav>

File: index.html [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;400;500;700;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="wrapper">
+    
+    
+      <nav class="navbar">
+
+        <header class="navbar-brand"><a href="home-page-link">Abenezer</a></header>
```
Comment (huseyinbiyik) [inline index.html]: As the previous reviewer requested, please follow this structure for semantic reasons.
```
<header>
  <a>logo</a>
  <nav>
    <ul>
      <li><a href="#link">Link</a></li>
    </ul>
  </nav>
</header>
```

File: index.html [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+          <li class="navbar-item">Contact</li>
+          <li class="navbar-item"><img src="./img/666162.png" alt="" class="nav-icon"></li>
+        </ul>
+      <div class="navbar-toggle">
+        <i class="fa-solid fa-bars"></i>
+      </div>
+      </nav>
+      <section class="header">
+      <div class="header-text"><h1>Hey There. <span>I'm Abenezer</span> </h1>
+      </div>
```
Comment (huseyinbiyik) [inline index.html]: The goal of this project is to code the given design pixel perfectly like maybe you have seen on some job ads. Sı please stick to the Figma design as the previous reviewer requested and fix the font-size and font-weight like on the below.
Screenshot 2023-01-19 at 7 29 09 PM
Screenshot 2023-01-19 at 7 30 16 PM

File: index.html [hunk 8]
```diff
@@ -0,0 +1,264 @@
...
+      <div class="body-text-header"><h2>I'm a Software Developer</h2>
```
Comment (huseyinbiyik) [inline index.html]: Kindly fix the font-family and font-size as the previous reviewer requested.
Screenshot 2023-01-19 at 7 31 00 PM
Screenshot 2023-01-19 at 7 40 46 PM

File: index.html [hunk 9]
```diff
@@ -0,0 +1,264 @@
...
+      </div>
+      <div class="body-text-main">
+        <p>I can help you build a product,feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
```
Comment (huseyinbiyik) [inline index.html:37]: Kindly stick to the Figma template and fix the font-family and size of the paragraph.

Screenshot 2023-01-19 at 7 32 39 PM
Screenshot 2023-01-19 at 7 33 28 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @Sofinaye,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Linter set up well and checks pass 🟢
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
❗️❗️ Important note: your PR/branch includes the codes belonging to the future branches. Please remove the unrelated code otherwise, your code will be invalidated for your future pull requests, because they will not be able to select the lines they want to review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 10]
```diff
@@ -0,0 +1,263 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio | Abenezer Fekadu</title>
+  <link rel="stylesheet" href="./style.css">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;400;500;700;800&display=swap" rel="stylesheet">
+</head>
```
Comment (Qoosim) [inline index.html]: - [ ] Roboto and Poppins google fonts used in the Figma design is not among the google fonts you imported. You need to import them before they could be used. At this moment, the font-family for the headline subtitle and paragraph does not have effect on the text. Kindly include them and use them in a proper way for them to take effect on those texts.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,263 @@
...
+          <li class="navbar-item">Portofolio</li>
+          <li class="navbar-item">About</li>
+          <li class="navbar-item">Contact</li>
+          <li class="navbar-item"><img src="./img/666162.png" alt="" class="nav-icon"></li>
+        </ul>
+      
+      </nav>
+      </header>
+      <section class="header">
+      <div class="header-text"><h1>Hey There. <span>I'm Abenezer</span> </h1>
```
Comment (Qoosim) [inline index.html]: - [ ] The font-size for the main title on your page is too large in oppose to what we have on the figma design. You can adjust the font-weight and font-size of your title to match what is on the guiding template.

### Your Design
your-main-title

### Figma Design
temp-main-title

- [ ] The hamburger icon is also missing on the right side of your header. Kindly fix that too.

### Your Design
your-hamburger

### Figma Design
temp-hamburger

File: index.html [hunk 12]
```diff
@@ -0,0 +1,263 @@
...
+      </div>
+      <div class="body-text-header"><h2>I'm a Software Developer</h2>
```
Comment (Qoosim) [inline index.html]: - [ ] The `Roboto` font-family you used does not exit in your impoted google fonts, so it falls back to the callback `serif` font-family after it. You need to include `Roboto`  font from google fonts before you can use  and it work. Kindly fix that.

File: index.html [hunk 13]
```diff
@@ -0,0 +1,263 @@
...
+      </div>
+      <div class="body-text-main">
+        <p>I can help you build a product,feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
```
Comment (Qoosim) [inline index.html:37]: - [ ] The same thing goes for this too. Kindly import the `Poppins` font and use it properly.

File: index.html (hunk 3 above)
Comment (Qoosim) [reply index.html]: - [ ] At this moment, your hamburger is missing on the page. Kindly look at that and fix.

### Your Design
your-hamburger

### Figma Design
temp-hamburger

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @Sofinaye, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuhaib042 — APPROVED** [review]
Hi @Sofinaye >, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approve
## Highlights
No linter errors. ✔️
Meaningful commit messages ✔️
Correct GitHub Flow. ✔️

## Optional suggestions
- [x] Kindly consider adding a Title heading in your pull request along with the summary to make it more professional.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**